mitodl/open-discussions
# Red dot next to incomplete profiles

The changes in #708 should be merged first so that the check for an existing profile image can be updated.

#### What are the relevant tickets?
Closes #680

#### What's this PR do?
Displays a red dot next to the profile image if user is logged in and certain profile fields (name, bio, headline, or image) are blank.

#### How should this be manually tested?
- Add `FEATURE_PROFILE_UI=True` to your `.env` file.
- Log in as a user with empty/null values for any of the following: bio, headline, name, image_small.  A red dot should appear at upper right of your profile image (or default avatar image).
- Using django shell or API request, insert values for all the above fields.  The red dot should no longer appear near your profile image.
